### DISCUSSION TITLE
Drop list for adding clients

### DISCUSSION THREAD

**Discussion by argie-w** [AdguardTeam/AdGuardHome#3055] [Ideas]
When using AdGuard Home as the DHCP Server it should be easier to add clients by either including a button next to their entry in the DHCP list or having a drop down list showing all mac addresses when adding them as a client.

This would make setting up the clients a lot easier than having to locate each device and copy the Mac addresses one by one. It is slow and painful.

Thanks

PS. Also why does DHCP give out addresses in order and not randomly?

**Comment by ameshkov** [comment]
Good idea, thank you.

> PS. Also why does DHCP give out addresses in order and not randomly?

Does it matter?

**Reply by argie-w** [reply]
No i guess it doesn’t really matter. I’m just used to getting random IP addresses.
